packages/daemon/src/guest.js

```diff
@@ -68,6 +52,7 @@ export const makeGuestMaker = ({
     const { petStore } = mailbox;
     const directory = makeDirectoryNode(petStore);
 
+    const { reverseIdentify } = specialStore;
```

Comment:
Incidentally, I needed the base case. I hope this doesn’t overlap too much with `followReverseLocate`. I am in favor of consolidating this into `reverseLocate` eventually.